derive-impl/src/compile_bytecode.rs

```diff
@@ -44,6 +44,11 @@ enum CompilationSourceKind {
     Dir(PathBuf),
 }
 
+struct FrozenModule {
```

Comment:
This struct is sharing the name `FrozenModule` with frozen_lib::FrozenModule but its code object is different. giving another name will be good to reduce later confusion.